### PR TITLE
SecureDrop 2.4.2

### PR DESCRIPTION
**Pull request by legoktm**
## Status

Ready for review, needs prod signed tag

## Description of Changes

Refs #6507.

## Checklist
* [ ] targets `release/2.4.2` branch
* [ ] CI passes
* [ ] updates correct files (compare with 0bdc72286ccf96f2413b52e031234c412aaf088c)

### CODE REVIEW

File: install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal [hunk 1]
```diff
@@ -1,8 +1,8 @@
-securedrop-app-code (2.4.2~rc1+focal) focal; urgency=medium
+securedrop-app-code (2.4.2+focal) focal; urgency=medium
 
   * see changelog.md
 
- -- SecureDrop Team <securedrop@freedom.press>  Wed, 03 Aug 2022 14:26:30 -0400
+ -- Kunal Mehta <legoktm@debian.org>  Mon, 08 Aug 2022 21:54:36 -0400
```
Comment (zenmonkeykstop) [inline install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal]: Should be updated to "SecureDrop Team <securedrop@freedom.press>" - historically we haven't used personal attributions in the changelog.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]
One change to the ID line in the changelog, otherwise looking good!

File: install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal (hunk 1 above)
Comment (legoktm) [reply install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal]: Gah, not sure why I didn't notice this last night. Fixed!

**Review by zenmonkeykstop — APPROVED** [review]
LGTM, can merge once CI passes.

**Pull request merged**